cooltools/cli/call_dots.py

```diff
@@ -108,60 +123,60 @@
     show_default=True,
 )
 @click.option(
-    "-v", "--verbose",
-    help="Enable verbose output",
-    is_flag=True,
-    default=False
-)
-@click.option(
-    "-s", "--output-scores",
-    help="At the moment it is a redundant option that"
-    " does nothing. Reserve it for a better dump"
-    " of convolved scores.",
-    type=str,
-    required=False,
+    "-v", "--verbose", help="Enable verbose output", is_flag=True, default=False
 )
 @click.option(
-    "--output-hists",
-    help="Specify output file name to store"
-    " lambda-chunked histograms. [Not implemented yet]",
-    type=str,
-    required=False,
-)
-@click.option(
-    "-o", "--output-calls",
+    "-o",
+    "--output-calls",
     help="Specify output file name where to store"
     " the results of dot-calling, in a BEDPE format."
     " Pre-processed dots are stored in that file."
     " Post-processed dots are stored in the .postproc one.",
     type=str,
 )
-@click.option(
-    "--score-dump-mode",
-    help="Specify file format for the dump of convolved scores."
-    " This dump is used for the downstream processing and"
-    " is read twice. Now 'parquet' is the only supported"
-    " format. 'cooler' and 'hdf' in the future.",
-    type=str,
-    default="parquet",
-    show_default=True,
-)
-@click.option(
-    "--temp-dir",
-    help="Create temporary files in specified directory.",
-    type=str,
-    default=".",
-    show_default=True,
-)
-@click.option(
-    "--no-delete-temp",
-    help="Do not delete temporary files when finished.",
-    is_flag=True,
-    default=False,
-)
+### Unused options commented out. TODO: remove them or implement
+# @click.option(
+#     "-s", "--output-scores",
+#     help="At the moment it is a redundant option that"
+#     " does nothing. Reserve it for a better dump"
+#     " of convolved scores.",
+#     type=str,
+#     required=False,
```

Comment:
let's remove them all - they are more technical/debugging options - that rely on a "single-pass" scenario - i.e. when we store results of the convolution on disk (abandoned for now at least)